deploy/openshift/kiali.yaml

```diff
@@ -83,6 +83,7 @@ spec:
       serviceAccount: kiali
       containers:
       - image: ${IMAGE_NAME}:${IMAGE_VERSION}
+        imagePullPolicy: ${PULL_POLICY}
```

Comment:
I think we want what Matt did before - he had something like ${IMAGE_PULL_POLICY} whose value is the entire string.

This way, if you don't set it, this is just empty and will just pick up the k8s default (and thus we are backwards compatible with our current getting started instructions and we don't have to worry about helm chart/ansible updates right now.

Reply:
see https://github.com/kiali/kiali/pull/470/files#diff-47101f922225a5e44356824f41906c92R86